.github/workflows/pull_request.yml

```
@@ -5,6 +5,7 @@ on:
     branches:
       - "master"
       - "branch-v*"
+      - "bug_fix_*"
```

Should this be `bug-fix-`?
```suggestion
      - "bug_fix_*"
```

---

I will ask this question in the session, thanks.